kernel_hardening_checker/checks.py

```diff
@@ -249,6 +246,27 @@ def add_kconfig_checks(l, arch):
         l += [AND(KconfigCheck('self_protection', 'kspp', 'INTEL_IOMMU', 'y'),
                   iommu_support_is_set)]
 
+    # CPU side-channels mitigations were renamed in 6.9
+    if VersionCheck((6, 9, 000)):
```

Comment:
Hello @jvoisin,

Thanks for the pull request!

Here we can do the job easier, without checking the version.

We can add a check of this type: `OR(new_name, old_name)`.

See the example commit https://github.com/a13xp0p0v/kernel-hardening-checker/commit/756c7103cd8f010f401556abd66da4f9e0b4b914.

The verbose output for Linux v6.8.9:
```
-------------------------------------------------------------------------------------------------------------------------
    <<< OR >>>                                                                             | OK: CONFIG_PAGE_TABLE_ISOLATION is "y"
CONFIG_MITIGATION_PAGE_TABLE_ISOLATION  |kconfig|     y      |defconfig | self_protection  | FAIL: is not found
CONFIG_PAGE_TABLE_ISOLATION             |kconfig|     y      |defconfig | self_protection  | OK
-------------------------------------------------------------------------------------------------------------------------
```